RobotCode/snobot2018/build.gradle

```diff
@@ -21,7 +21,7 @@ plugins {
 
 apply plugin: com.snobot.simulator.plugin.SnobotSimulatorPlugin
 snobotSimVersions {
-    snobotSimVersion = "0.7.3"
+    snobotSimVersion = "0.9.0"
```

Comment:
simulator update.  Everyone should run that dumb batch script `generate_ide_files` I made because I made the wrong choice on project management